libs/accounts/recovery-phone/src/lib/recovery-phone.service.ts

```diff
@@ -66,6 +66,12 @@ export class RecoveryPhoneService {
     }
 
     const code = await this.otpCode.generateCode();
+    await this.recoveryPhoneManager.storeUnconfirmed(
```

Comment:
Not sure why but I wasn't seeing this show up in Redis, it looked like there was an entry but nothing to expand/look at. When I'm done with this PR I'll circle back and see if it works removing the change in this file.